best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
```

Comment:
My pet peeve: contexts nested a bazillion deep. They make specs unreadable and difficult to refactor.